src/libraries/System.Private.CoreLib/src/System/Threading/SemaphoreSlim.cs

```diff
@@ -459,7 +459,7 @@ private bool WaitUntilCountOrTimeout(int millisecondsTimeout, uint startTime, Ca
                     }
                 }
                 // ** the actual wait **
-                bool waitSuccessful = Monitor.Wait(m_lockObjAndDisposed, remainingWaitMilliseconds);
+                bool waitSuccessful = OperatingSystem.IsBrowser() ? false : Monitor.Wait(m_lockObjAndDisposed, remainingWaitMilliseconds);
```

Comment:
This isn't safe.  It's the same problem I outlined in ManualResetEventSlim.

Reply:
marking this API and the one in ManualResetEventSlim.cs as Unsupported on browser causing all Wait methods in Task.cs and all Async Read/Write/Copy methods in Stream to be annotated as Unsupported. Which further causing many derived Streams, and Task-related types annotated, hope that is what we need